doc/source/tutorials.rst

```diff
@@ -0,0 +1,256 @@
+
+   counts = AWS.execute_circuit(c, nshots=1000).frequencies()
+   print(counts)
+
+
+Submit a circuit in verbatim mode
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In verbatim mode, the circuit is executed on the device without any transpilation. The user has to ensure that the circuit is specifically written in the device's native gates and gates obey the topology of the device.
+Therefore, before submitting a Qibo circuit in verbatim mode, it is recommended to extract the Amazon Braket device's information. We will demonstrate this below.
+
+Extracting Amazon Braket device parameters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+More information on IQM Garnet device can be found here: https://aws.amazon.com/braket/quantum-computers/iqm/.
+
+To use BraketClientBackend, we import these packages first.
+
+.. code-block:: python
+
+   from braket.aws import AwsDevice
+   from braket.devices import Devices, LocalSimulator
+   from qibo_cloud_backend.braket_client import BraketClientBackend
+
+The qubit connectivity on the IQM Garnet device can be visualised using networkx. We will also extract the device's native gates.
+
+.. code-block:: python
+
+   import networkx as nx
+
+   device = "arn:aws:braket:eu-north-1::device/qpu/iqm/Garnet"
+   connectivity_graph = AwsDevice(device).properties.paradigm.connectivity.connectivityGraph
+   native_gates = AwsDevice(device).properties.paradigm.nativeGateSet
+   print(native_gates)
+
+   G = nx.Graph()
+   for node, neighbors in connectivity_graph.items():
+      for neighbor in neighbors:
+         G.add_edge(node, neighbor)
+
+   nx.draw(G, pos=nx.spring_layout(G), with_labels=True, node_color='lightblue', node_size=500, font_size=10, font_weight='bold', edge_color='gray')
+
+
+Submit the circuit
+~~~~~~~~~~~~~~~~~~
+
+Let us run a circuit with `verbatim_circuit=True` on an Amazon Braket device, using IQM Garnet as an example. Here, when `verbatim_circuit=True`, the circuit is submitted as is onto the Amazon Braket device. The device expects to receive a circuit that is written in native gates with qubits in the range of the device. For IQM Garnet, the native gates are `CZ` and `PRX` gates. IQM Garnet has qubits indexed from 1 to 20.
+
+.. code-block:: python
+
+   from qibo import gates, Circuit as QiboCircuit
+   import numpy as np
+
+   c = QiboCircuit(5)
+   c.add(gates.PRX(1, 0.5*np.pi, 1.5*np.pi))
+   c.add(gates.PRX(4, 0.142857142857143*np.pi, 0))
+   c.add(gates.CZ(4, 1))
+   c.add(gates.PRX(1, 0.5*np.pi, 0.5*np.pi))
+   c.add(gates.M(1))
+   c.add(gates.M(4))
+
+   print(c.draw())
+
+We should get this circuit:
+
+.. code-block:: python
+
+   q0: -------------
+   q1: -prx-Z-prx-M-
+   q2: -----|-------
+   q3: -----|-------
+   q4: -prx-o-M-----
+
+Since IQM Garnet has qubits indexed from 1 to 20, we will intentionally leave qubit `q0` empty without any gates. An error will be raised if there are gates on any qubits not in the range from 1 to 20.
```

Comment:
Oh. I used qubits `q1` and `q4` because they are coupled (see image of the chip's connectivity [here](https://www.iqmacademy.com/qpu/garnet/). I will just paste the image to save you one click: :-D
![Screenshot 2024-09-23 at 6 04 34 PM](https://github.com/user-attachments/assets/2d545f1d-c2e4-4ad2-94a6-68870aaa4051)

If we submit this circuit with `verbatim_circuit = True`
``` python
   q0: -prx-Z-prx-M-
   q1: -prx-o-M-----
```
there will be an error message because `q0` and `q1` are not connected. But `q1` and `q2` will work. As for using `q1` and `q4`, I've found that the qubit indices need to correspond to the topology, and that creating a circuit with 5 qubits works. So if we were to use the coupled qubits `q19` and `q20`, assuming that 19 and 20 have the lowest error rates etc.., then we'll need to create a 21 qubit circuit.

That said, would you know if it is possible to create a qibo circuit like this, without having to create a 21 qubit circuit?
``` python
   q19: -prx-Z-prx-M-
   q20: -prx-o-M-----
```
If it's possible, then I think running it in verbatim mode on the Amazon Braket device might be doable!

Reply:
Ah ok I see, so here you are running with `verbatim=True`. Then maybe, try to include the plot of the topology in the tutorial and explicitely explain that since `verbatim=True` you have to create the circuit this way